src/main/java/uk/ac/ebi/ampt2d/accession/server/FileAccessionResponse.java

```diff
@@ -15,31 +15,30 @@
  * limitations under the License.
  *
  */
+package uk.ac.ebi.ampt2d.accession.server;
 
-package uk.ac.ebi.ampt2d.accession.test;
+import uk.ac.ebi.ampt2d.accession.file.UuidFile;
 
-import uk.ac.ebi.ampt2d.accession.AccessionRepository;
-
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
 
-public class TestMapAccessionRepository implements AccessionRepository<String> {
-    private Map<String, String> storage = new HashMap<>();
-
-    @Override
-    public Map<String, String> get(List<String> objects) {
-        Map<String, String> accessions = new HashMap<>();
-        for (String object : objects) {
-            if (storage.containsKey(object)) {
-                accessions.put(object, storage.get(object));
-            }
-        }
-        return accessions;
+public class FileAccessionResponse {
```

Comment:
This class was already in place, but do we really need it? We could just map the objects to the response straight away. We would need to remove the `@JsonIgnore` annotation from the accession field though.